tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
 
     public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
```

Comment:
I know, do you mean our IDE will dynamically add new server type even if customer won't upgrade IDE package?